Packs/PasswordResetViaChatbot/README.md

```diff
@@ -0,0 +1,22 @@
+
+Remember that time when couldn't login to your system because you forgot your password? Ever needed the help of IT just after they left for lunch?
```

Comment:
```suggestion
Remember that time when you couldn't log in to your system because you forgot your password? Ever needed the help of IT just after they left for lunch?
```